provider/googleProvider.js

```diff
@@ -92,6 +92,9 @@ class GoogleProvider {
           },
           additionalProperties: false,
         },
+        cloudFunctionSecretEnvironmentVariables: {
+          type: 'object',
```

Comment:
```suggestion
          type: 'object',
          properties: {
              key: {
                  type: 'string',
                  minLength: 1,
              },
              projectId: {
                  type: 'string',
                  minLength: 1,
              },
              secret: { type: 'string' },
              version: {
                  type: 'string',
                  pattern: '^(latest|[0-9\.]+)$',
              },
          },
          required: ['key', 'sercet', 'version'],
          additionalProperties: false,
```

In the docs it is `projectId`, not `project`.

Not sure about `version` pattern, because I'm not sure if version numbers can be decimals. Also, docs say it is a `string` so assuming if version is numeric then it is passed as a string, not a number.

And big disclaimer here: I don't know what version / flavour of JSONSchema is being used here 😄